cmd/lotus-gateway/api.go

```diff
@@ -172,6 +177,14 @@ func (a *GatewayAPI) StateWaitMsg(ctx context.Context, msg cid.Cid, confidence u
 	return a.api.StateWaitMsgLimited(ctx, msg, confidence, stateWaitLookbackLimit)
 }
 
+func (a *GatewayAPI) WalletVerify(ctx context.Context, k address.Address, msg []byte, sig *crypto.Signature) (bool, error) {
```

Comment:
I believe WalletVerify should already be working in lite-mode. Is this needed for something else?
Here's where WalletVerify is exercised in the gateway tests:
https://github.com/filecoin-project/lotus/blob/feat/gateway-wallet-verify/cmd/lotus-gateway/endtoend_test.go#L72

Reply:
Yes. lotus-gateway is for quite a bit more than just lotus-lite.

Reply:
it makes sense that it works via lotus-lite because the verification is done in the wallet code itself